src/internal/tabbable.ts

```diff
@@ -58,7 +58,7 @@ function isTabbable(el: HTMLElement) {
   }
 
   // At this point, the following elements are considered tabbable
-  return ['button', 'input', 'select', 'textarea', 'a', 'audio', 'video', 'summary'].includes(tag);
+  return ['button', 'input', 'select', 'textarea', 'a', 'audio', 'video', 'summary', 'iframe'].includes(tag);
```

Comment:
Nice catch. Did updating this on its own have any affect?

Reply:
Yes. It actually allowed focusing into iframes.

Reply:
Did we try this on a video element? Because the initial problem was with a video element, not an iframe.

Reply:
Yes, video and audio.